### ISSUE TITLE
[Feature Request] Option to hide account icon/picture and country from the main page

### ISSUE THREAD

**Issue by rpgdev** [yuliskov/SmartTubeNext#1694]
Simply that: please add a "misc" option to hide (not simply autohide as it does now) the account icon/picture and country from the main page, it's overloading it.

**Comment by Jamie-Landeg-Jones** [comment]
Yes please. I really dislike that big flag that now appears since the last upgrade.

**Comment by rpgdev** [comment]
@Jamie-Landeg-Jones yep, the flag it's not even high quality, looks blurry.. I actually downgraded because of it. It's version 14.8X for me now and I will be a little bit more circumspect about updating from now on unless I have to.. I'm sure there have been many improvements since but hard to notice what I'm missing other than that giant eyesore.

**Comment by CoadyKidd** [comment]
This is already an option I was able to do it
You go to 
Settings
General
Buttons at the top of main window 
Then u can turn them off
